src/to-be-made-a-dependency/filtered-list.ts

```diff
@@ -0,0 +1,188 @@
+import {
```

Comment:
maybe move this to src/components ?

Reply:
https://github.com/openscd/oscd-filtered-list